README.md

```diff
@@ -1,99 +1,104 @@
 # LBS-server
 
 # Installation Guide
+
 ## Build and Run
-### Step 1. Clone the github repo to your prefered directory using
-```
-git clone https://github.com/Suwi-inc/LBS-server.git
-``` 
-### Step 2. Inside the repository directory install the project dependencies
+
+### Step 1. Clone the github repo to your preferred directory
 ```sh
-poetry install
+git clone https://github.com/Suwi-inc/LBS-server.git
+cd LBS-server
 ```
-### Step 3: Create .env
 
-The project depends on some config that it expects to be provided from a .env file at the `LBS` directory. 
+### Step 2. Install Docker if it is not installed
+https://docs.docker.com/engine/install/
+
+### Step 3: Create .env file
+
+The project depends on some config that it expects to be provided from an .env file at the `LBS` directory.
 ```sh
 cd LBS
-``` 
-Here is the expected content of the file
-```env
-DB_URI='Postgres_DB_connect_string_here'
-SECRET_KEY='your_auth_secret'
 ```
-### Step 4: Run Migrations
-Inside the `LBS` directory run
+
+Here is the expected content of the file:
 ```sh
-poetry run flask db init
-poetry run flask db migrate
-poetry run flask db upgrade
+SECRET_KEY='your_auth_secret'
```

Comment:
What is this secret?

Reply:
I don't know. As I understand, it is only required for generating jwt. There is also this field in `.env` file. And there is no problem, it they are not equal.

Reply:
Then learn what it is and write it in the README. It makes 0 sense to write something into README if you have no idea what it is.